release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -120,6 +120,12 @@ The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-w
 [id="ocp-4-7-scale"]
 === Scale
 
+[id="ocp-4-7-scale-new-vrf-cni-plugin"]
+==== New VRF CNI plugin allows secondary networks to be assigned to different VRFs
+
+A new VRF CNI plugin is now available which allows you to assign additional networks to network interfaces. When you specify an additional network using a `rawCNIConfig` configuration for the CNO custom resource, the CNO creates the `NetworkAttachmentDefinition` custom resource automatically.
```

Comment:
I'd reword this in a way where we associate additional networks (which we are already able to create) to a vrf using the extra configuration.
In this way, it looks like the VRF plugin is the one allowing the additional networks, whereas it just associates them to a VRF.